Tools/Driver.AliCloud.props

```diff
@@ -0,0 +1,93 @@
+<!--
```

Comment:
As mentioned in the last review, we are certainly appreciative of AliCloud contributing this enhancement, but why do we need to check in AliCloud specific driver properties?

or did you intend for these to be an example?